model/vo/TemplateVO.php

```diff
@@ -215,6 +219,30 @@ public function setTaskStoryId($taskStoryId) {
         $this->taskStoryId = $taskStoryId;
     }
 
+    public function getInitTime(): ?int {
```

Comment:
@jaragunde Instead of documenting the function I thought of types notation for the parameters and return values like in this line, what do you think?
I find it more readable and it also makes the code more reliable - it will throw warnings and errors if we use pass wrong types

If we are using PHP 7.4 in the prod we can also type the properties making the code even cleaner: https://wiki.php.net/rfc/typed_properties_v2

Reply:
I agree with using the types notation in new code. Thanks for the suggestion!

About typed properties, current minimum PHP version is 7.0, we will start requiring PHP 7.3 in the pull request #499. We could also bump it to 7.4, as it is widely available nowadays, but let's do that separately.